src/Grid.js

```diff
@@ -165,7 +169,7 @@ Grid.prototype.redraw = function(newOptions, callback) {
   var items;
 
   window.requestAnimationFrame(function() {
-    if (this.columns !== newOptions.columns) {
+    if (this.columns !== newOptions.columns || this.columnClasses !== newOptions.columnClasses) {
```

Comment:
As you stated before, this can, and I believe it should, be controlled by CSS using media queries so why add the complexity here.
I am happy to merge without this feature. 👍